src/pages/Groups/fines/FineItem.tsx

```diff
@@ -108,31 +111,35 @@ const FineItem = ({ fine, groupSlug, isAdmin, hideUserInfo, ...props }: FineItem
       <Collapse in={expanded}>
         <Divider />
         <Stack gap={1} sx={{ p: [1, undefined, 2] }}>
-          <div>
-            {fine.reason && <Typography variant='subtitle2'>{`Begrunnelse: ${fine.reason}`}</Typography>}
-            <Typography variant='subtitle2'>{`Opprettet av: ${fine.created_by.first_name} ${fine.created_by.last_name}`}</Typography>
-            <Typography variant='subtitle2'>{`Dato: ${formatDate(parseISO(fine.created_at), { fullDayOfWeek: true, fullMonth: true })}`}</Typography>
-          </div>
+          <Stack direction={{ xs: 'column', md: 'row' }} gap={1}>
+            {fine.reason && <Chip icon={<HelpIcon />} label={fine.reason} variant='outlined' />}
+            <Chip icon={<GavelIcon />} label={`${fine.created_by.first_name} ${fine.created_by.last_name}`} variant='outlined' />
+            <Chip icon={<TimeIcon />} label={`${formatDate(parseISO(fine.created_at), { fullDayOfWeek: true, fullMonth: true })}`} variant='outlined' />
+          </Stack>
+          <Stack direction={{ xs: 'column', md: 'row' }} gap={1}>
+            <Chip icon={<ApprovedIcon />} label={fine.approved ? 'Godkjent' : 'Ikke godkjent'} variant='outlined' />
+            <Chip icon={<PaidIcon />} label={fine.payed ? 'Betalt' : 'Ikke betalt'} variant='outlined' />
+          </Stack>
```

Comment:
Chip for status på betaling og godkjent-status er en god idé! Men å bruke chips til den andre informasjonen er ikke særlig lurt. Det er ikke intuitivt at chip med navn er den som opprettet bot, og reason kan potensielt være langt og passer dermed ikke i en chip. I tillegg er det nok fint med en farge på de 2 chip'ene som blir stående igjen
```suggestion
          <Stack direction='row' gap={1}>
            <Chip color={fine.approved ? 'success' : 'error'} icon={<ApprovedIcon />} label={fine.approved ? 'Godkjent' : 'Ikke godkjent'} variant='outlined' />
            <Chip color={fine.payed ? 'success' : 'error'} icon={<PaidIcon />} label={fine.payed ? 'Betalt' : 'Ikke betalt'} variant='outlined' />
          </Stack>
          <div>
            {fine.reason && <Typography variant='subtitle2'>{`Begrunnelse: ${fine.reason}`}</Typography>}
            <Typography variant='subtitle2'>{`Opprettet av: ${fine.created_by.first_name} ${fine.created_by.last_name}`}</Typography>
            <Typography variant='subtitle2'>{`Dato: ${formatDate(parseISO(fine.created_at), { fullDayOfWeek: true, fullMonth: true })}`}</Typography>
          </div>
```